README.md

```diff
@@ -49,6 +50,35 @@ Particle orientations are stored as
 [`scipy.spatial.transform.Rotation`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html) objects.
 These transformations should rotate the basis vectors of a reference such that they are correctly oriented in a tomogram.
 
+## `ProjectionModel`
+A `ProjectionModel` is a [pandas `DataFrame`](https://pandas.pydata.org/docs/) with specific column 
+headings for the parameters of a projection model. Together, this information constitues a 'tilt-series alignment'.
+
+| Heading         | Python name   | Semantics                                  |
+|:----------------|:--------------|:-------------------------------------------|
+| `rotation_x`    | ROTATION_X    | tomogram rotation around x-axis            |
+| `rotation_y`    | ROTATION_Y    | tomogram rotation around y-axis            |
+| `rotation_z`    | ROTATION_Z    | tomogram rotation around z-axis            |
+| `dx`            | SHIFT_X       | particle shift in x-dimension              |
+| `dy`            | SHIFT_Y       | particle shift in y-dimension              |
```

Comment:
I'm a bit confused by these headers. This is about tilt images right? so not about tomograms or particles?

Reply:
sorry, forgot to update the shifts labels in the table copied from poses! 

The way rotations are parametrised for tilt-series alignments in all softwares (AFAIK) is in terms of 'rotations of the specimen' i.e. in the microscope reference frame

Reply:
ok, makes sense!